SafeguardSessions/README.md

```diff
@@ -2,14 +2,14 @@
 
 ## Table of Contents
 
-- [About the project](#about-the-project)
-- [Installation and usage](#installation-and-usage)
-- [Common errors](#common-errors)
-- [Troubleshooting](#troubleshooting)
-- [Release policy](#release-policy)
-- [Contribution](#contribution)
-- [License](#license)
-- [References](#references)
+- [About the project]
```

Comment:
I know sections are not final, I just put a mark here so that we don't forget to update the table of contents with the new section(s).

Reply:
Done.